### PR TITLE
[plugin.video.gamekings] 1.2.20

### PR DESCRIPTION
**Pull request by skipmodea1**
### Description
v1.2.20 (2023-01-15)
- fixed some youtube-id's by removing leading/trailing spaces
- fixed playing of videos using MPEG-DASH (kudo's to peak3d)
- updated name
- updated logo
- less logging
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by skipmodea1** [comment]
@enen92 Thanks for looking at the matrix PR so fast. Could you have a look at this one also if possible? 

Much obliged :)

File: plugin.video.gamekings/resources/lib/gamekings_play.py [hunk 1]
```diff
@@ -120,21 +120,32 @@ def playVideo(self):
                             # check that 'login_error' is in the response. If that's the case, the login was not ok
                             # and the username and password in settings are not ok.
                             if str(html_source).find('login_error') >= 0:
+
+                                log("Login Error!", "login was NOT succesfull!")
```
Comment (enen92) [inline plugin.video.gamekings/resources/lib/gamekings_play.py]: You have typos in successful

File: plugin.video.gamekings/resources/lib/gamekings_play.py [hunk 2]
```diff
@@ -120,21 +120,32 @@ def playVideo(self):
...
+
                                 xbmcgui.Dialog().ok(LANGUAGE(30000), LANGUAGE(30601), LANGUAGE(30602),
                                                     LANGUAGE(30603))
                                 sys.exit(1)
                             else:
                                 # dialog_wait.create("Login Successfull", "Currently looking for video")
 
-                                log("self.video_page_url", "login was succesfull!!")
+                                log("Login", "login was succesfull!!")
```
Comment (enen92) [inline plugin.video.gamekings/resources/lib/gamekings_play.py]: Same

**Review by enen92 — APPROVED** [review]

**Comment by skipmodea1** [comment]
@enen92 i fixed the typos.

**Pull request merged**